doc/src/manual/faq.md

```diff
@@ -250,8 +252,8 @@ julia> threearr()
 
 ### [What does "type-stable" mean?](@id man-type-stability)
 
-It means that the type of the output is predictable from the types of the inputs.  In particular,
-it means that the type of the output cannot vary depending on the *values* of the inputs. The
+It means that the return type is predictable from the types of the inputs.  In particular,
```

Comment:
I think `type of the output` is better here because it refers to the type of the specific value that is returned. Return type evokes the inferred type, which isn't quite what's meant here.